src/entities/User/routes.ts

```diff
@@ -0,0 +1,124 @@
+import { WithAuth, auth } from 'decentraland-gatsby/dist/entities/Auth/middleware'
+import handleAPI from 'decentraland-gatsby/dist/entities/Route/handle'
+import routes from 'decentraland-gatsby/dist/entities/Route/routes'
+import { hashMessage, recoverAddress } from 'ethers/lib/utils'
+import { Request, Response } from 'express'
+import isEthereumAddress from 'validator/lib/isEthereumAddress'
+
+import { DiscourseService } from '../../services/DiscourseService'
+import isDebugAddress from '../Debug/isDebugAddress'
+
+import { GATSBY_DISCOURSE_CONNECT_THREAD, MESSAGE_TIMEOUT_TIME } from './constants'
+import UserModel from './model'
+import { ValidationMessage } from './types'
+
+export default routes((route) => {
+  const withAuth = auth()
+  route.get('/is-profile-validated/:address', handleAPI(isProfileValidated))
```

Comment:
What if this is a `/profile` endpoint instead that we could check if a property exists or has a certain value to see if is validated? We can call this in a `useProfile` hook.